src/module_PI/Raspberry_PI/main/Camera.java

```diff
@@ -0,0 +1,64 @@
+package module_PI.Raspberry_PI.main;
+
+import java.awt.Dimension;
+import java.io.File;
+import java.io.IOException;
+
+import javax.imageio.ImageIO;
+
+import com.github.sarxos.webcam.*;
+import com.github.sarxos.webcam.ds.v4l4j.V4l4jDriver;
+
+public class Camera {
+	
+	private String filePath;
+	private Webcam webcam;
+	private Dimension[] imageResolutions = new Dimension[] {
+			WebcamResolution.PAL.getSize(),
+			WebcamResolution.HD720.getSize(),
+			new Dimension(2000, 1000),
+			new Dimension(1000, 500),
+};
+	public Camera(){
+		this("./");
+	}
+	
+	public Camera(String filePath){
+		setFilePath(filePath);
+		setWebcamDriver();
+		buildWebcam();
+	}
+	
+	private void setWebcamDriver(){
+	    Webcam.setDriver(new V4l4jDriver());
+	}
+	
+	private void buildWebcam(){
+		webcam = Webcam.getDefault();
+		Webcam webcam = Webcam.getDefault();
```

Comment:
You are constructing another local webcam object here. This makes it difficult to know whether your `setCustomViewSizes` and `setViewSize` are being set on the `webcam` that is a part of the _class_ or whether it is being set on the `webcam` local to this _method_ (which will fall out of scope and be eligible for garbage collection after you leave this method).

I think the intention here was to set these things on the _class_ `webcam` so I would say that this line needs to be deleted.